lldb/tools/lldb-dap/DAP.h

```diff
@@ -219,6 +218,9 @@ struct DAP {
   llvm::StringSet<> modules;
   /// @}
 
+  /// Number of lines of assembly code to show when no debug info is available.
+  static constexpr uint32_t number_of_assembly_lines_for_nodebug = 32;
```

Comment:
Constants use a `k_` prefix.
```suggestion
  static constexpr uint32_t k_number_of_assembly_lines_for_nodebug = 32;
```